hugo/layouts/partials/home/hero.html

```diff
@@ -0,0 +1,27 @@
+<header class="home-hero">
+    <div class="home-hero__wrap">
+        <div class="home-hero__info">
+            <h1>{{ .title }}</h1>
+            <p>{{ .desc }}</p>
+            <div class="home-hero__info__links">
```

Comment:
styles are css matters... 
If html is the same, it can be used a partial, and the css of the roadmap and hero will override the buttons styles for each component.

aside: what are the differences? because they looks the same...